srcpkgs/signond/template

```diff
@@ -1,19 +1,20 @@
 # Template file for 'signond'
 pkgname=signond
-version=8.59
+version=8.60
 revision=1
+wrksrc="signond-VERSION_${version}-4d195e4dc7a47ff5cb51e36a83d4d05808c5befe"
 build_style=qmake
-short_desc="Daemon providing SSO over D-Bus for the gSSO framework"
-maintainer="John Rowley <enterthevoid@codesector.co>"
-license="LGPL-2.1"
-hostmakedepends="pkg-config doxygen"
-makedepends="qt5-devel"
-wrksrc="${pkgname}-VERSION_${version}-14f058c36208a551c80d0e98d76164fb87b2b8af"
 configure_args="LIBDIR=/usr/lib"
 conf_files="/etc/signond.conf"
+hostmakedepends="pkg-config doxygen"
+makedepends="qt5-devel"
+short_desc="Daemon providing SSO over D-Bus for the gSSO framework"
+maintainer="John Rowley <enterthevoid@codesector.co>"
+license="LGPL-2.1-only"
 homepage="https://gitlab.com/accounts-sso/signond"
-distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>${pkgname}-${version}.tar.gz"
-checksum=23f6658bf358cc09b858402ee0857db2a2809276625eb01673a0a77969740762
+distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>signond-${version}.tar.gz"
+checksum=a9cd90cf3e8b1a8bdbe5acf46c797a817092e77946c69fc3ed3a76930f3a76aa
+nocross=qmake
```

Comment:
Can you provide more details?

Reply:
Yes, I care.
Most templates with `build_style=qmake` add everything to `hostmakedepends` as well to fix this.
I think adding the headers to the include path is the better solution.

Reply:
unbroken

Reply:
Thank you. Any chance to fix armv7?

Reply:
It was a random failure. Now passed with no changes.